app/components/open_graph_tags_component.rb

```diff
@@ -0,0 +1,21 @@
+class OpenGraphTagsComponent < ViewComponent::Base
+  def title
+    "railsdevs.io"
+  end
+
+  def description
+    "Find Ruby on Rails developers looking for freelance and full-time work."
```

Comment:
Not a biggy, but this tag line differs slightly from the [README](https://github.com/joemasilotti/railsdevs.io/blame/main/README.md#L3).

> Find Rails developers looking for freelance and contract work.

On a related note: If the site is also intended for developers who are currently employed full-time, it may be worth having an availability duration option (as well as the current `available_on` date). 

An availability duration would provider developers, who are currently employed full-time, the ability to represent their notice period without continually having to update the `available_on` date. 

Perhaps the availability option is determined by if a developers current employment is either full-time vs freelance or contractor. That said, I believe contractors tend to have an end date, thus availability duration may be more suitable.